de-de/geaenderte-optionen.md

```diff
@@ -0,0 +1,89 @@
+# Geänderte Spieloptionen
+
+*Fabulously Optimized* ändert manche Standard-Optionen von *Minecraft*, um dein Spielerlebnis zu verbessern.
+
+Weil das Modpack *YOSBR* ("Your Options Shall Be Respected", "Deine Einstellungen sollten respektiert werden") nutzt, werden mögliche Einstellungen in der `options.txt` oder `config`-Ordner statt den Standardeinstellungen von *Fabulously Optimized* genommen.
+
+Das heißt, dass du das Modpack immer updaten kannst, ohne deine Einstellungen zu verlieren!
+
+### Einstellungen
+
+| Einstellung                 | Beschreibung                                                                                                                                                                                                                                                        | Vanilla  | Modpack | Grund für die Änderung                                                                                                                                                         |
+| --------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| autoJump                    | Springt automatisch springen, wenn man sich einem 1-Block hohem Block nähert.                                                                                                                                                                                       | an       | aus     | Im normalen Spiel (Vanilla) ist es für die meisten eher ein Hindernis als eine nützliche Funktion, weshalb viele diese Einstellung deaktivieren.                               |
```

Comment:
Maybe "true (an)" and "false (aus)" would be better as it does refer to the actual options.txt values
same for "movement"/"none" below

Reply:
:+1: makes sense!